samples/Exporters/Exporters.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
this was part of another PR, was it? Can you please rebase

Reply:
ok, done